src/js/modules/test/counterlike.test.js

```diff
@@ -13,15 +13,15 @@ test('should have 9 items', () => {
     item_id: '65494',
   }, {
     item_id: '46881',
-  },{
+  }, {
     item_id: '1',
-  },{
+  }, {
     item_id: '12',
-  },{
+  }, {
     item_id: '32',
-  },{
+  }, {
     item_id: '67',
-  },{
+  }, {
```

Comment:
- [ ] in the test you are not actually testing a function that you are using in your application you are just testing the mock function which is not the goal of testing. part of the reason for this is that there is no specific function counting the likes.